frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/noobaa-sso.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { checkErrors } from '../../../integration-tests-cypress/support';
+
+describe('Check noobaa link in obejct service dashboard and perform SSO', () => {
+  before(() => {
+      cy.install();
+    cy.login();
```

Comment:
You should have login() first and then install()
```suggestion
    cy.login();
    cy.install();
```